src/my_project/tagging/image_scoring.py

```
@@ -0,0 +1,149 @@
+    def score_candidate_images(self, possibles: list[Candidate]) -> list[Candidate]:
+        scored_candidates = []
+        images: list[BytesIO] = []
+        with ThreadPoolExecutor(max_workers=5) as executor:
+            images = list(
+                executor.map(
+                    self.http.download_img,
+                    [struct.issue.image.medium_url for struct in possibles],
+                )
+            )
+        for index, entry in enumerate(possibles):
+            logger.info(f"Started processing the cover of {entry.issue.volume.name}.")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)